packages/runtime/lib/runtime.js

```diff
@@ -62,16 +71,17 @@ class Runtime extends EventEmitter {
     this.#runtimeTmpDir = getRuntimeTmpDir(configManager.dirname)
     this.#runtimeLogsDir = runtimeLogsDir
     this.#env = env
-    this.#services = new Map()
+    this.#workers = new RoundRobinMap()
```

Comment:
Why is this needed here?

Reply:
This map serves two purposes:

1. Track all the existing workers.
2. Implement a round-robin get, which is used in `#getWorkerById`.